pyr2.html

```diff
@@ -58,31 +56,38 @@
 var a13 = qnbank[Qline][7];
 var snod = qnbank[Qline][8];
 var a14 = qnbank[Qline][9];
-document.write("<DL>");
+
+document.innerHTML="<DL>";
 document.write("<table border=0 width=550>");
-document.write("<tr>");
+document.innerHTML="<tr>";
 document.write("<td width=100% align=left>");
-document.write("<DT>", qn);
-document.write("<br><br>");
-document.write("<DD>",snoa,'. ', a11);
+document.innerHTML="<p> Burger           </p>";
+document.innerHTML="<DT>";
+document.write(qn);
+document.write("<br>");
+document.write(snoa,'. ', a11);
 document.write("<br>");
 document.write(snob,'. ', a12);
 document.write("<br>");
 document.write(snoc,'. ', a13);
 document.write("<br>");
 document.write(snod,'. ', a14);
-document.write(" </td>");
-document.write(" </tr>");
+document.innerHTML="</DT>"
+document.write("</td>");
+document.write("</tr>");
 document.write("</table>");
 document.write("</DL>");
 document.write("<tr>");
 document.write("<td width=100% align=right> Answer. ");
-document.write('<input type="text" name="textbox" size="1">');
+document.write('<input type="text" name="textbox" size="1" maxlength="1" onblur = "giveBoxValue(this.value)">');
```

Comment:
Use `addEventListener` instead of event properties. Decouples the JS from the HTML to allow easier refactoring and debugging.